assets/wizards/readerRevenue/index.js

```diff
@@ -203,29 +214,10 @@ class ReaderRevenueWizard extends Component {
 					label: __( 'Address', 'newspack' ),
 					path: '/location-setup',
 				},
-				{
-					label: __( 'Stripe', 'newspack' ),
-					path: '/stripe-setup',
-				},
-				{
-					label: __( 'Platform', 'newspack' ),
-					path: '/',
-					exact: true,
-				},
+				...platformAgnosticFields,
 			];
 		} else if ( NRH === platform ) {
-			return [
-				{
-					label: __( 'Settings', 'newspack' ),
```

Comment:
Don't we still need to be able to access the "Settings" tab in the dashboard to enter `NRH Organization ID` and `NRH Salesforce Campaign ID` options, if NRH is the platform but _not_ using the streamlined donation block?

Reply:
Whoops, that's true. Re-added and changed tab title to "NRH Settings" for clarity in 5b162874ca54f500da929883aeb32f6860d4bb99

Reply:
This looks great! But now I want the "Stripe" tab to be labelled "Stripe Settings" for consistency? 😬

Reply:
Yeah, that makes sense - added inf2abcba346b07b7ef716da71b5bfb2726b49ff37